velox/exec/fuzzer/FuzzerUtil.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/exec/Split.h"
+
+using namespace facebook::velox;
+
+const std::string kHiveConnectorId = "test-hive";
+
+std::vector<exec::Split> makeSplits(
```

Comment:
Can you also change joinFuzzer to use this? Thanks!

Reply:
I tried, but then found in JoinFuzzer
```
std::shared_ptr<connector::ConnectorSplit> JoinFuzzer::makeSplit(
```

so one option is to also move it to FuzzerUtil, but I'm not sure if they're general enough to be moved here, what do you think?

Reply:
Hmm, why join fuzzer needs a different version? Can you help take a look? Thanks!